src/screens/login/styles.ts

```diff
@@ -0,0 +1,45 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+  loginContainer: {
+    marginTop: 32,
+  },
+  title: {
+    color: '#090909',
+    fontSize: 32,
+    fontWeight: 'bold',
+  },
+  inputLabel: {
+    color: '#323232',
+    fontSize: 16,
+    marginBottom: 8,
+  },
+  inputContainer: {
+    marginVertical: 32, // Space between title
```

Comment:
Aqui com esse comentário dá pra levantar uma discussão interessante: se esse estilo for reutilizado no futuro, pode ser que ele não esteja abaixo de um "title", e portanto ou o comentário ou a margem não fariam muito sentido, né?

Quando você não quer que o espaçamento de um componente seja o mesmo em todos os lugares em que ele é utilizado, é melhor que o "pai" desse componente cuide desse espaçamento. Faz sentido?